README.md

```diff
@@ -1,7 +1,7 @@
 Visual Studio Codespaces Quickstart Sample
 ======================================
 
-This is a sample project that supports the Visual Studio Codespaces quickstart documentation. It can be used for either the [browser based quickstart](https://aka.ms/vso-docs/quickstart/browser) or the [Visual Studio Code based quickstart](https://aka.ms/vso-docs/quickstart/vscode).
+This is a sample project that supports the Visual Studio Codespaces quickstart documentation. It can be used for either the [browser based quickstart](https://docs.microsoft.com/en-us/visualstudio/codespaces/how-to/browser) or the [Visual Studio Code based quickstart](https://aka.ms/vso-docs/quickstart/vscode).
```

Comment:
Thanks for noticing this.

I think the link should go to the quickstart rather than the how-to:

https://docs.microsoft.com/visualstudio/codespaces/quickstarts/browser

Also it's good practice to remove 'en-us' so that readers will get the localized version.

Reply:
I've updated the URL to the one you mentioned in the comment.